services/wex/secure_token.md

```diff
@@ -0,0 +1,37 @@
+# Secure token
+
+When you start a new registration or renew an existing one a token is created that is then used to identify the in progress `transient_registration`. From a design point of view this delivers a number of benefits to the user (as long as they retain the link)
```

Comment:
```suggestion
When you start a new registration, or start to renew an existing one, a token is created. We use this token to identify the in-progress `transient_registration`. From a design point of view, this delivers a number of benefits to the user (as long as they retain the link):
```

Maybe a bit nitpicky, but I thought the first sentence was pretty complex, so had a go at breaking it up to make it a little easier to read.